rre-dataset-generator/src/test_env/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  solr:
+    image: solr:9.8.1
+    container_name: solr_test
+    ports:
+      - "8983:8983"
+    volumes:
+      - ./volumes/solr:/var/solr
+      - ./solr/data:/opt/solr/data
+    command: ["solr-precreate", "testcore"]
+
+  dataset-loader:
+    image: curlimages/curl:8.14.1
+    depends_on:
+      - solr
+    volumes:
+      - ./solr/data:/data
```

Comment:
The folder `solr/data` is shared between the two containers, which is not recommended